src/esp/gfx/PTexMeshShader.h

```diff
@@ -27,11 +27,12 @@ class PTexMeshShader : public Magnum::GL::AbstractShaderProgram {
 
   explicit PTexMeshShader();
 
-  PTexMeshShader& bindTexture(Magnum::GL::Texture2D& texture,
-                              uint32_t textureUnit = 0) {
-    texture.bind(textureUnit);
-    return *this;
-  }
+  // ======== texture binding ========
+  // Note: the texture binding points are explicitly specified in .cpp
```

Comment:
This looks like a comment that should go in the implementation file. Someone looking at the header to understand the API wouldn't a priori have any expectations of explicit binding point specification or care about GL version constraints, and this would be confusing text, in my opinion